frontends/web/src/components/guide/guide.tsx

```diff
@@ -37,13 +37,19 @@ export const store = new Store<SharedProps>({
     guideExists: false,
 });
 
-apiGet('config').then(({ frontend }) => {
-    if (frontend && frontend.guideShown != null) { // eslint-disable-line eqeqeq
-        store.setState({ shown: frontend.guideShown });
-    } else {
-        store.setState({ shown: true });
-    }
-});
+// if apiGet() is invoked imediatelly this can error due to cyclinc dependencies
+// request.js:64 Uncaught TypeError: Cannot read properties of undefined
+// (reading 'runningInQtWebEngine')
+// TODO: this should probably be in a context
+setTimeout(() => {
+    apiGet('config').then(({ frontend }) => {
+        if (frontend && frontend.guideShown != null) { // eslint-disable-line eqeqeq
```

Comment:
the warning could be removed/fixed? i assume it's about `!=` vs `!==`?

Reply:
I'll change it to `!== undefined`